app/controllers/api/v0/pages_controller.rb

```diff
@@ -191,13 +191,13 @@ def filter_maintainers_and_tags(collection)
   def format_page_json(page)
     page['maintainers'] = page.delete('maintainerships').collect do |item|
       item['maintainer']
-        .reject {|k, _| k == 'created_at' || k == 'updated_at'}
+        .reject {|k, _| ['created_at', 'updated_at'].include?(k)}
```

Comment:
I did a performance/allocation pass on it, extracting the array of values. I also changed them to `!` methods because it looked like the hashes could be updated in place rather than needing to be duped. e.g. `merge` (which returns a new copy of the hash) vs `merge!` (which updates in place)